jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -33,7 +33,6 @@ class DeploymentFlow extends React.Component {
       deploying: false,
       credential: this.props.credential,
       loadingTerms: false,
-      loggedIn: this.props.isLoggedIn(),
```

Comment:
If the logged in state is set here when the component is created then the `isLoggedIn()` method never gets called again to update the state.